tech-committee-charter.md

```diff
@@ -61,6 +61,34 @@ The development process will include a process for the TC to oversee and approve
 
 The TC and entire technical community will follow any processes as may be specified by the Cloud Native Computing Foundation relating to the intake and license compliance review of contributions, including the CNCF IP Policy.
 
+### Sponsorship Requirements
+
+The TC will establish and maintain sponsorship across the SIGs and subprojects of OpenTelemetry.
+
+SIGs will be classified into three classifications of Sponsorship requirement:
+
+* *Escalation Sponsorship*: The TC sponsor will be a primary means of escalation of cross-project concerns from the SIG. This communication is primarily
+  offline, e.g. via CNCF slack or issue communication. The TC sponsor will participate in [GC check-ins](gc-check-ins.md) with SIG leadership.
+  Additionally, the TC will aide the SIG in developing road maps during this check-in, and aligning to overall OpenTelemetry goals.
```

Comment:
```suggestion
  Additionally, the TC will aid the SIG in developing roadmaps during this check-in, and aligning to overall OpenTelemetry goals.
```

Reply:
You decide which one is better since your English is way better than me (English is my 2nd language). I asked Google AI and got this suggestion:

![image](https://github.com/user-attachments/assets/88f1d93c-5381-4fee-a672-bcd3227747e4)

Reply:
I just meant I had it as `roadmap` and the build failed.  It's not MY choice, it's our spellcheck tool.

Reply:
I am surprised that the spell checker has issues with "roadmap", but if this persists, then you can add it to the wordlist and no longer worry about it

Reply:
@svrnm Where is the wordlist?  I'll update it there.

Reply:
https://github.com/open-telemetry/community/blob/main/.cspell.yaml

Reply:
updated the dictionary and use roadmaps now.  Thanks @svrnm !